README.md

```diff
@@ -86,20 +84,124 @@ code because of all the basic column-renamings and the full listing of health co
 side, we've been told by DAP leadership that the HLES structure is effectively immutable on their end,
 so it's unlikely we'll ever need to rewrite large chunks of this processing logic.
 
-### Hack: Uploading Results to Terra
-The TDR will eventually be fully integrated with Terra Workspaces. In the immediate future, however,
-there's no "golden path" for bridging data from the repository into the workspace model. DAP's near-term
-users aren't very technical, so it's unlikely they'll be comfortable figuring things out on their own.
+## Additional Datasets
 
-To give ourselves some breathing room, we've written a [script](./hack/convert-output-to-tsv.py) to convert
-the outputs of our transformation pipeline into workspace-compatible TSVs. We can manually run the script
-after the two Dataflow pipelines as a final transformation step, then upload the outputs to a workspace.
+### CSLB Data
+The Canine Social and Learned Behavior is a followup survey to HLES that is administered annually. 
+The goal of this survey is to assess age-related cognitive and behavioral changes in dogs. The first 
+time a participant fills out the CSLB survey, a baseline score for  is established. Participants will 
+have the opportunity to complete the survey again every year. This repeated design allows us to learn 
+how the dogs in the study change over time. Answers to all questions are required.
+
+#### CSLB Extraction
```

Comment:
🔧 This section seems to cover just the _criteria_ for extraction, what do you think about updating the title to reflect that?